packages/concerto-cto/lib/parser.pegjs

```diff
@@ -726,6 +725,66 @@ relative_part
 absolute_URI
   = scheme ":" hier_part ("?" query)?
 
+/**
+ * SemVer.org v2
+ * https://semver.org/spec/v2.0.0.html
+ */
+semver
+  = versionCore:versionCore
+    pre:('-' @preRelease)?
+    build:('+' @build)?
```

Comment:
Can you add a test for a version string that uses preRelease and build components, please?